src/averageAges.js

```diff
@@ -15,47 +15,62 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let males = people.filter(({ sex }) => sex === 'm');
+
+  if (century) {
+    males = males.filter(male => Math.ceil(male.died / 100) === century);
+  }
+
+  const averageAges = males.map(
+    male => {
+      return male.died - male.born;
```

Comment:
this can also be global variable 
`const calculateAge = (person) => person.died - person.born;`

Reply:
yes, let's create a dedicated function up above to use whenever we'd like to